ccdproc/tests/test_image_collection.py

```diff
@@ -85,6 +85,13 @@ def test_summary_is_summary_info(self, triage_setup):
             location=triage_setup.test_dir, keywords=['imagetyp', 'filter'])
         assert img_collection.summary is img_collection.summary_info
 
+    def test_filenames_are_set_properly(self, triage_setup):
+        fn = ['filter_no_object_bias.fit', 'filter_object_light_foo.fit']
+        img_collection = image_collection.ImageFileCollection(
+            location=triage_setup.test_dir, filenames=fn, keywords=['filter'])
+        assert img_collection.files is fn
+        assert img_collection.summary is img_collection.summary_info
```

Comment:
This test is not needed because they are always equal. `summary_info` is just the deprecated property.